test/parallel/test-net-write-fully-async-buffer.js

```diff
@@ -11,14 +11,14 @@ const data = Buffer.alloc(1000000);
 
 const server = net.createServer(common.mustCall(function(conn) {
   conn.resume();
+  server.close();
 })).listen(0, common.mustCall(function() {
   const conn = net.createConnection(this.address().port, common.mustCall(() => {
     let count = 0;
 
     function writeLoop() {
       if (count++ === 200) {
```

Comment:
```suggestion
      if (count++ === 20) {
```

For consistency, but we can do it in another PR.